### DISCUSSION TITLE
Inserting `\usepackage{xparse}` before citeproc command definition in LaTeX preamble

### DISCUSSION THREAD

**Discussion by wuffi** [jgm/pandoc#9973] [Q&A] [answered]
Is there a way to include a LaTeX package before citeproc-related definitions in the preamble of LaTeX output generated by *Pandoc*:  

Running `pandoc` with the `citeproc` option puts a `\NewDocumentCommand\citeproc{}{}` definition into the preamble of the LaTeX output.  
The macro `\NewDocumentCommand` is defined by the `xparse` LaTeX package (but also directly in the LaTeX kernel of newer LaTeX versions). 
The `xparse` package gets not included in the preamble with my versions of *Pandoc* and *Citeproc* and thus compilation with LaTeX fails.
Including LaTeX packages is easy with Pandoc, but they would only appear after `citeproc`-related definitions in the preamble of the `*.tex` file generated by *Pandoc*.

**Comment by badumont** [comment, marked as answer]
You can always use a custom LaTeX template: run `pandoc -D latex` to get the default template used by pandoc, modify it, and use your custom version with the --template command line option.

Older LaTeX installations could be made to work out of the box by adding to the template something like:
***@***.***{NewDocumentCommand}{\usepackage{xparse}}{}`.

**Reply by wuffi** [reply]
Thanks, using a custom template is probably indeed the cleanest solution.
I was a bit afraid and thus hesitant to do so, but starting from Pandoc's default one, it is quite feasible and actually easy to adjust it.
